testing/src/scenario/state.py

```diff
@@ -865,13 +865,17 @@ class CheckInfo(_max_posargs(1)):
     level: pebble.CheckLevel | None = None
     """Level of the check."""
 
+    startup: pebble.CheckStartup = pebble.CheckStartup.ENABLED
+    """Startup mode of the check."""
+
     status: pebble.CheckStatus = pebble.CheckStatus.UP
     """Status of the check.
 
     :attr:`ops.pebble.CheckStatus.UP` means the check is healthy (the number of
     failures is fewer than the threshold), :attr:`ops.pebble.CheckStatus.DOWN`
     means the check is unhealthy (the number of failures has reached the
-    threshold).
+    threshold), and :attr:`ops.pebble.CheckStatus.INACTIVE` means the check is
+    not currently running.
```

Comment:
It's a tad unclear to me whether "not currently running" means:
- disabled, or
- last check finished, next check not started yet

Reply:
> It's a tad unclear to me whether "not currently running" means:
> 
> * disabled, or
> * last check finished, next check not started yet

A check never "finishes" (individual changes or tasks might), it's always running (unless it's stopped). I'll try to tweak the wording without diverging from the Pebble terms or saying "inactive means inactive" :)